css/main.scss

```diff
@@ -446,29 +446,21 @@ pre[class=highlight] {
     margin: 0;
 }
 
-.nav .nav-link {
-	color: #2753e3 !important;
-  }
-
 .nav .nav-link:hover {
 	color: #1034a8 !important;
 }
 
 a {
-	color: #2753e3;
+	color: #216fc9;

```

Comment:
These changes might be needed still to fix contrast a11y issues.

Reply:
These are your changes merging into the published branch